src/runCompletion.ts

```diff
@@ -1,4 +1,5 @@
 import { CancellationToken, Position, Range, TextDocument } from "vscode";
+import * as vscode from "vscode";
```

Comment:
there is a named import from vscode one line above, just add the `window` and the `Uri` to it